File: Arrays/Repeating And Missing Number/Brute/code.java
public class code {
    public static int[] findNumbers(int[] nums) {
        int n = nums.length;
        int repeating = -1, missing = -1;

        for(int i=1;i<=n;i++){
            int count = 0;
            for(int j=0;j<n;j++){
                if(nums[j] == i) count++;
            }
            if(count == 0) missing = i;
            if(count > 1) repeating = i;
        }
        return new int[]{repeating, missing};
    }

    public static void main(String[] args) {
        int[] nums = {3,1,2,5,3};
        int[] ans = findNumbers(nums);
        System.out.println("Repeating: " + ans[0] + ", Missing: " + ans[1]);
    }
}

File: Arrays/Repeating And Missing Number/Brute/code.py
def findNumbers(nums):
    n = len(nums)
    repeating, missing = -1, -1

    for i in range(1, n+1):
        count = nums.count(i)
        if count == 0:
            missing = i
        if count > 1:
            repeating = i
    return repeating, missing

nums = [3,1,2,5,3]
print("Repeating:", findNumbers(nums)[0], "Missing:", findNumbers(nums)[1])
